webpack.config.js

```diff
@@ -5,7 +5,10 @@ module.exports = {
   devServer: {
     proxy: {
       '/api': 'http://localhost:3001'
-    }
+    },
+    headers: {
+      "Access-Control-Allow-Origin": "*"
+    },
```

Comment:
fixes chrome for me. not sure how the CORS errors are happening only in my chrome but 🤷‍♂